client/electron/src/setup.ts

```diff
@@ -201,17 +201,30 @@ export class ElectronCapacitorApp {
     };
   }
 
+  setAppMenu(): void {
+    if (process.platform === 'darwin') {
+      const menu = new Menu();
+      menu.append(new MenuItem({
+        label: app.name,
+        submenu: [
+          {
+            label: 'Quit',
```

Comment:
`Quit` should be translated depending on the locale (and updated when the locale changes). See what's done in `updateConfig()` for the tray.